.github/workflows/docs.yml

```diff
@@ -79,7 +79,7 @@ jobs:
 
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@v4
-        if: ${{env.OK_TO_DEPLOY_DOCS}} == 'true' && github.ref == 'refs/heads/master'
```

Comment:
@heerener not sure it should have been  `${{env.OK_TO_DEPLOY_DOCS== 'true' && github.ref == 'refs/heads/master'}}` or we shouldn't use `${{ }}` at all

Reply:
If it works like this, better leave it like this then!